app/assets/javascripts/TortoiseJS/agent/component/input.coffee

```diff
@@ -127,8 +127,13 @@ window.RactiveInput = RactiveWidget.extend({
     """
     {{>input}}
     {{>contextMenu}}
+    {{# widget.boxedValue.type === 'Color' || widget.boxedValue.type === 'Number' }}
     <editForm idBasis="{{id}}" boxtype="{{widget.boxedValue.type}}"
+              display="{{widget.display}}" />
+    {{ else }}
+        <editForm idBasis="{{id}}" boxtype="{{widget.boxedValue.type}}"
               display="{{widget.display}}" isMultiline="{{widget.boxedValue.multiline}}" />
+    {{/}}
```

Comment:
Hey Jason, thanks for the tip, I can definitely do it that way.  And I actually did try to get the work pointed back at master before pushing it up for the PR, but that obviously didn't work.  I'll get that resolved, too.